.github/workflows/multi-platform-docker-build.yml

```diff
@@ -0,0 +1,110 @@
+name: crtsh-exporter-multi-platform-docker-build
+
+on:
+  push:
+    # tags:
+    #   - "v*.*.*" 
+    branches:
+      - master
+
+  schedule:
```

Comment:
Why run on a schedule rather than trigger on commits?

Reply:
It's true that the unless the underlying source code changes, it's pointless to run periodic pipelines. 

We use a lot of container security scanners on our K8s clusters and they're always complaining about containes OS dependencies are vulnerable. 

These pipelines only produce docker images, so periodic runs will ensure updated container OS